gui/webdisplay/src/RWebWindowsManager.cxx

```diff
@@ -212,7 +213,7 @@ std::map<std::string, std::string> RWebWindowsManager::GetServerLocations()
    TString cfg = gEnv->GetValue("WebGui.ServerLocations","");
    auto arr = cfg.Tokenize(";");
    if (arr) {
-      TIter next(arr);
+      TIter next(static_cast<const TCollection *>(arr));
```

Comment:
Is this still needed? I would expect the include to already solve the problem, no?

Reply:
IOW `const TObjArray *` should automatically cast to `const TCollection *` once the compiler sees the declaration (by including the correct header)